testER/src/App.jsx

```diff
@@ -357,32 +342,40 @@ const App = (props) => {
     // toggle off all tracks??
   }
 
-  return (
-    <>
-      <div id='map-container' onKeyDown={logKey} onKeyUp={logKey}>
-        {/* <a href='https://earthranger.com/'>
+  return <TrackContext.Provider value={{ displayTracks, setTracks, tracks }}>
```

Comment:
`<TrackContext.Provider>` provides values for children consumers to receive. In this, it's the three parts of your track state management needed by the TrackButton (and possibly other future consumers): the method for changing track visibility on the map, the method for setting the tracks in state, and the value of the track state.